pages/thc/[processRef]/review.tsx

```diff
@@ -121,25 +129,18 @@ const ReviewPage: NextPage = () => {
       pausable
     >
       <React.Fragment>
+        <TenancySummary
+          details={{
+            address: address.result,
+            tenants: allTenantNames,
+            tenureType: tenureType.result,
+            startDate: tenancyStartDate.result,
+          }}
+        />
```

Comment:
Does it make semantic sense to have two lists? I like that we reuse `TenancySummary`, but it doesn't feel like it's correct to have two lists.

Maybe we could add an `extraRows` prop to `TenancySummary`?

Reply:
Yeah, might be a better idea

Reply:
I've given this a go, but I'm not 100% convinced about my implementation - if you could take a look and let me know if I've got the wrong end of the stick, that would be great!